diff --git a/dist/types/overrideannotation.d.ts b/dist/types/overrideannotation.d.ts
index 8545f6bc4bdabe295d8973792b0e32c20319454f..815e9d99a21d464f5a506d09698bb43639af7543 100644
--- a/dist/types/overrideannotation.d.ts
+++ b/dist/types/overrideannotation.d.ts
@@ -1,4 +1,4 @@
 import { Annotation } from "../internal";
-import type { ClassMethodDecorator } from "./decorator_fills";
-export declare const override: Annotation & PropertyDecorator & ClassMethodDecorator;
+import type { ClassGetterDecorator, ClassMethodDecorator } from "./decorator_fills";
+export declare const override: Annotation & PropertyDecorator & ClassMethodDecorator & ClassGetterDecorator;
 export declare function isOverride(annotation: Annotation): boolean;
